src/commands/math/basicSymbols.js

```diff
@@ -140,7 +140,7 @@ var BuiltInOpNames = {}; // http://latex.wikia.com/wiki/List_of_LaTeX_symbols#Na
 var TwoWordOpNames = { limsup: 1, liminf: 1, projlim: 1, injlim: 1 };
 var AutoOpNames = Options.p.autoOperatorNames = { _maxLength: 9 };
 (function() {
-  var mostOps = ('arg deg det dim exp gcd hom inf ker lg lim ln log max min sup'
+  var mostOps = ('arg deg det dim exp gcd hom inf ker lg ln log max min sup'
```

Comment:
so in your approach what does just plain typing `l`, `i`, and `m` do? They remain italicized, like 3 variables?